blog/templates/blog_post.haml

```diff
@@ -16,6 +16,12 @@
     %meta{:property => "twitter:image", :content => "{{ URL_PREFIX }}://{{ request.META.HTTP_HOST }}{{ photo.url }}"}
 {% endblock %}
 
+{% block extra_head %}
+    %script{:async => "async",
+            :src => "https://platform-api.sharethis.com/js/sharethis.js#property=5f18836b2a81520019e1054d&product=inline-share-buttons",
```

Comment:
Is there any changes to get specific version of this buttons? We would like to have SRI up here

Reply:
Not really no. ☹️ This link is most likely generated and linked with an account I created. You can customize those buttons on their website and the changes automatically appear on our page. Therefore I don't believe it would be possible to get an SRI.